core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -23,19 +23,33 @@ import (
 type logPollerWrapper struct {
 	utils.StartStopOnce
 
-	routerContract      *functions_router.FunctionsRouter
-	pluginConfig        config.PluginConfig
-	client              client.Client
-	logPoller           logpoller.LogPoller
-	subscribers         map[string]evmRelayTypes.RouteUpdateSubscriber
-	activeCoordinator   common.Address
-	proposedCoordinator common.Address
-	blockOffset         int64
-	nextBlock           int64
-	mu                  sync.Mutex
-	closeWait           sync.WaitGroup
-	stopCh              utils.StopChan
-	lggr                logger.Logger
+	routerContract            *functions_router.FunctionsRouter
+	pluginConfig              config.PluginConfig
+	client                    client.Client
+	logPoller                 logpoller.LogPoller
+	subscribers               map[string]evmRelayTypes.RouteUpdateSubscriber
+	activeCoordinator         common.Address
+	proposedCoordinator       common.Address
+	requestBlockOffset        int64
```

Comment:
Would it make sense to group this and the next 3 variables in a struct as "LogPollerConfig"?

Reply:
That might be a good suggestion. I'd certainly appreciation Bolek's thoughts here.  It might be nice to establish some sort of "standard" for how we organize and validation our growing list of configuration options.

Reply:
That being said, I really want to get this merged today, so I am hoping any changes we consider to be limited for this PR.

Reply:
@justinkaseman I had a sync w/ @bolekk this morning and we don't yet have a single place where we validate config, nor a standard structure for config settings.  Therefore, I'll punt on this problem given the intended narrow scope of this PR. Maybe we can make a backlog ticket for better config structure & validation later.